src/pal/src/loader/module.cpp

```diff
@@ -220,7 +222,7 @@ LoadLibraryExW(
         return NULL;
     }
     
-    CHAR lpstr[MAX_LONGPATH];
+    CHAR * lpstr = NULL;
```

Comment:
I think you wanted to pass the multibyteLibraryPath by a pointer to it and make it an output parameter.